planner/core/logical_plan_test.go

```diff
@@ -938,7 +938,7 @@ func (s *testPlanSuite) TestPlanBuilder(c *C) {
 		},
 		{
 			sql:  "show columns from t where `Key` = 'pri' like 't*'",
-			plan: "Show->Sel([eq(cast(key), 0)])",
+			plan: "Dual->Sel([eq(cast(key), 0)])->Projection",
```

Comment:
This plan generated looks more weird than the old one?

Reply:
Changed. But since the show doesn't do pruningm, the project is remained.